website/views/pages/homepage.ejs

```diff
@@ -880,7 +881,7 @@
                 <img src="/images/icon-android-16x16@2x.png" alt="Android" >
                 <img src="/images/icon-ios-16x16@2x.png" alt="iOS">
               </div>
-              <div purpose="comparison-column" v-if="comparisonModeForSecurity === 'qualys'">
+              <div purpose="comparison-column" v-if="comparisonModeForSecurity === 'qualys', 'defender'">
```

Comment:
@mike-j-thomas Defender doesn't have the level of insight we (Fleet) has when it comes to a "Complete device inventory", would it make more sense to leverage "Mobile not supported" or add "Limited inventory available"?

Reply:
Mike M: "Basic info only" (Defender has a lot of breadth but limited depth.)